### PR TITLE
Add DBT theme

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Adds the DBT theme

### How to review
Test `test_theme_dbt` and `test_theme_dbt_ni` themes and check they have the correct logos

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by rmccar** [comment]
[Create new logo for DBT (1 SP)](https://trello.com/c/BMRQseEM/5637-create-new-logo-for-dbt-1-sp)

File: scripts/run_validator.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
-tag=latest
```
Comment (rmccar) [inline scripts/run_validator.sh:2]: Revert reminder

**Review by Yuyuutsu — COMMENTED** [review]
Tests look good, and I am able to see the DBT logo as well as DBT NI logos in the test schemas. Just one question, is the DBT logo meant to be the same size for both `Test_theme_db.json`  and `Test_theme_db_ni.json` because when opening the test schema for ` Northen Ireland` only, the logo is a lot bigger compared to when it is paired with another logo for example in `Test_theme_db_ni.json`.

**Comment by rmccar** [comment]
> Tests look good, and I am able to see the DBT logo as well as DBT NI logos in the test schemas. Just one question, is the DBT logo meant to be the same size for both `Test_theme_db.json` and `Test_theme_db_ni.json` because when opening the test schema for `Northen Ireland` only, the logo is a lot bigger compared to when it is paired with another logo for example in `Test_theme_db_ni.json`.

The DBT logo seems to be the same size for me in all schemas, would you be able to show me what you see?

**Comment by rmccar** [comment]
> Oh no I see that too, I just wanted to check if that was the case. Because with the Northern Ireland logo it changes sizes depending if it is by itself or not. For example the`test_theme_northenireland `the logo is bigger here.

Oh yeah thats intentional

**Review by Yuyuutsu — APPROVED** [review]
LGTM then !

**Comment by petechd** [comment]
Am I doing something wrong reviewing this or are these new logos not added to the `./templates/assets`?

**Review by petechd — APPROVED** [review]

**Pull request merged**